### PR TITLE
[#54] update README.md

### PR DESCRIPTION
**Pull request by puresin5117**
- remove old milestone

### CODE REVIEW

**Comment by ztkmkoo** [comment]
Closes #60

File: README.md [hunk 1]
```diff
@@ -17,14 +17,4 @@ https://github.com/ztkmkoo/dss/blob/develop/CODE_CONVENTION.md
 
 ## CODE OF CONDUCT
 
-https://github.com/ztkmkoo/dss/blob/develop/CODE_OF_CONDUCT.md
```
Comment (ztkmkoo) [inline README.md:20]: Not delete this..

Comment (puresin5117) [reply README.md:20]: Sorry. i will fix it.

Comment (puresin5117) [reply README.md:20]: Please check last line... the code is not deleted..

Comment (ztkmkoo) [reply README.md:20]: oh.. you are right...
i will merge it now

**Pull request merged**